### PR TITLE
stopgaps: detect and warn about row level security

### PR DESCRIPTION
**Pull request by eviefp**
Fixes https://github.com/prisma/schema-team/issues/390

### CODE REVIEW

**Review by tomhoule — APPROVED** [review]
nice!

File: schema-engine/sql-introspection-tests/tests/commenting_out/postgres.rs [hunk 1]
```diff
@@ -400,3 +400,50 @@ ALTER TABLE blocks
...
+        }
+
+        model foo {
+          id    Int    @id @default(autoincrement())
+          owner String @db.VarChar(30)
+        }
+    "#]];
+    api.expect_datamodel(&expected).await;
+    Ok(())
+}
```
[the remark this answers is not in the document]
Comment (Druue) [reply schema-engine/sql-introspection-tests/tests/commenting_out/postgres.rs:449]: Yeah, agreed. Unsure why we don't have a separate schema file that we can read from like we do elsewhere 🤔

**Review by Druue — APPROVED** [review]
LGTM ✨

File: schema-engine/sql-introspection-tests/tests/commenting_out/postgres.rs (hunk 1 above)
[the remark this answers is not in the document]
Comment (eviefp) [reply schema-engine/sql-introspection-tests/tests/commenting_out/postgres.rs:449]: Yeah, it's not even really commenting out but this is where our stopgap tests for warnings ended up, so ehh. It kinda still fits the general mood.

**Pull request merged**